app/src/main/java/com/andryoga/safebox/ui/view/login/LoginFragment.kt

```diff
@@ -44,6 +45,21 @@ class LoginFragment : Fragment() {
 
         setupObservers()
 
+        binding.ShowHintText.setOnClickListener {
+
+            if (binding.ShowHintText.text == getString(R.string.show_hint)) {
+
+                Utils.switchText(binding.ShowHintText, getString(R.string.hide_hint))
+                Utils.switchVisibility(binding.displayHintText)
+                Utils.switchVisibility(binding.hintDivider)
```

Comment:
No need to call this method 2 times, as this util method can have multi params:
try calling it like this : Utils.switchVisibility(binding.displayHintText, binding.hintDivider)